dev/parm/config/gcafs/config.base.j2

```diff
@@ -124,7 +124,7 @@ export BASE_ENV="${HOMEgfs}/env"
 export SDATE="{{ SDATE }}"
 export EDATE="{{ EDATE }}"
 export EXP_WARM_START="{{ EXP_WARM_START }}"
-export assim_freq=6
+export assim_freq="{{ assim_freq }}" # Assimilation frequency in hours
```

Comment:
Noting that there may be a few places that we assume `assim_freq` to be 6.  One of which is
https://github.com/NOAA-EMC/global-workflow/blob/f57dd0690a0019c1f2d4ee54e80b70ec97f91a3b/dev/workflow/setup_expt.py#L209

Though I don't think this particular case is an issue.

Reply:
Something to put in the icebox would be to remove all hard coded cases of this, as I anticipate in the not too distant future, hourly DA being a feature request for the workflow

Reply:
Opened https://github.com/NOAA-EMC/global-workflow/issues/3849 to tackle this at some point.